support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-id.html

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
This wrapper (`div`) is very helpful if you want to do changes in all help boxes.